ssha/ssh.py

```diff
@@ -10,15 +10,21 @@
 
 def _get_address(instance_ip):
 
-    username = config.get('ssh.username')
```

Comment:
This change is not taking into account how username is being generated:

https://github.com/claranet/ssha/blob/7de4afc5d51120f0ce14637aab877d37002e038e/ssha/config.py#L185-L191